q2_feature_table/__init__.py

```diff
@@ -20,8 +20,7 @@
 __version__ = get_versions()['version']
 del get_versions
 
-__all__ = ['rarefy', 'presence_absence', 'relative_frequency', 'summarize',
-           'merge', 'merge_seqs', 'filter_samples', 'filter_features',
-           'merge_taxa', 'tabulate_seqs', 'overlap_methods',
-           'core_features', 'group', 'heatmap', 'heatmap_choices',
-           'filter_seqs', 'subsample']
+__all__ = ['rarefy', 'presence_absence', 'relative_frequency', 'transpose',
+            'summarize','merge', 'merge_seqs', 'filter_samples', 'filter_features',
```

Comment:
Lint error (extra space)

Reply:
Thanks for the review, I will take care of these!